opensrp-pnc/src/main/java/org/smartregister/pnc/pojo/PncStillBorn.java

```diff
@@ -0,0 +1,22 @@
+package org.smartregister.pnc.pojo;
+
+public class PncStillBorn {
```

Comment:
this file is not necessary

Reply:
i have used this file in still birth section